packages/device-management-kit/src/api/transport/model/Errors.ts

```diff
@@ -39,8 +39,8 @@ export class NoAccessibleDeviceError extends GeneralDmkError {
 }
 
 export class OpeningConnectionError extends GeneralDmkError {
-  override readonly _tag = "ConnectionOpeningError";
-  constructor(readonly err?: unknown) {
+  override _tag = "ConnectionOpeningError";
```

Comment:
[COULD] Same with the class name like others.
```suggestion
  override _tag = "OpeningConnectionError";
```